tests/test_anc.py

```diff
@@ -41,6 +41,7 @@ def attended_anc_facility_proportion(
     return attended_facility_proportion
 
 
+@pytest.mark.skip(reason="Model 14 ANC updates have obsoleted this test.")
```

Comment:
@albrja how important is to update these tests? we can also talk in person at mid-day

Reply:
You don't have to update the tests if it is a lot of work. The whole test suite was my attempt to help resolve a lot of the previous v&v issues we had in the model. We could talk more about updating the tests to reflect your implementation but model tests in general are somewhat difficult to maintain.

I think you should avoid skip all tests because at that point you should remove them but I do think it is useful to have tests around that might offer some sort of validation or sanity check on certain components.